### ISSUE TITLE
contact module not working

### ISSUE THREAD

**Issue by therobyouknow** [NaturalHistoryMuseum/scratchpads2#6546]
Raised by admin of: https://insectafgseag.myspecies.info/

https://www.facebook.com/groups/665993564280327/posts/975084706704543/

**Comment by therobyouknow** [comment]
Resuming looking into this now.

**Comment by therobyouknow** [comment]
Did this ever work, I wonder?  If it has worked in the past and stopped working then that's a potentially useful line of enquiry.

**Comment by Archilegt** [comment]
Yes, it did work, at least until March 5th, 2021. Someone used the form to offer graphic design services to Myriatrix. See below.

On Fri, Mar 5, 2021 at 1:28 PM <[adi@ndmails.com](mailto:adi@ndmails.com)> wrote:

    Keith K. ([adi@ndmails.com](mailto:adi@ndmails.com)) sent a message using the contact form at
    http://myriatrix.myspecies.info/contact/1.

    Just wanted to ask if you would be interested in getting external help with
    graphic design? We do all design work like banners, advertisements, photo
    edits, logos, flyers, etc. for a fixed monthly fee.

    We don't charge for each task. What kind of work do you need on a regular
    basis? Let me know and I'll share my portfolio with you.

**Comment by therobyouknow** [comment]
Hello @Archilegt thank you - that is useful information.

I'm pleased to advise that the **password email facility** should now be working, following initial testing. More detail here: https://github.com/NaturalHistoryMuseum/scratchpads2/issues/6447#issuecomment-1138610498

Therefore it's possible that the contact form will also now work, because a submission into the contact form is relayed by our server to the admin's email. Now that some email changes fix the password reset email, it is possible that these same fixes resolve the contact form issue, too - as you suggested too.
